tools/proofers/dp_proof.js

```diff
@@ -72,103 +67,88 @@ function changeFontSize(font_size_index, font_size)
     fixText();
 }
 
-function showAllText()
```

Comment:
Did we drop this function intentionally?

Reply:
it was not referenced anywhere and eslint was complaining. It looks like it was added for debugging to be honest.